front/public/locales/fr/operationalStudies/scenario.json

```
@@ -70,7 +70,7 @@
   "toggleTimetable": "Basculer l'affichage de la grille horaires",
   "trainCount_one": "1/{{ totalCount }} trains",
   "trainCount_other": "{{ count }}/{{ totalCount }} trains",
-  "trainCount_zero": "0/{{ totalCount }} train",
+  "trainCount_zero": "0/{{ totalCount }} trains",
```

```suggestion
  "trainCount_zero": "0/{{ totalCount }} train",
```

In French, when the number is zero, the singular is used

---

The use of plural should apply based on the totalCount, not the filtered results of 0 or 1. 
For example, if we have 10 trains in total and the filter shows 0 invalid trains, it should display '0/10 trains' . 
'trainCount_zero' and 'trainCount_one' need to be adjusted to align with the total count, rather than just the filtered values

---

Oh ok, so no change needed, good job for this PR !